src/System.Runtime.Serialization.Json/tests/DataContractJsonSerializer.cs

```diff
@@ -1326,19 +1715,36 @@ private static T SerializeAndDeserialize<T>(T value, string baseline, DataContra
 
         using (MemoryStream ms = new MemoryStream())
         {
-            dcjs.WriteObject(ms, value);
-            ms.Position = 0;
+            try
+            {
+                dcjs.WriteObject(ms, value);
+                ms.Position = 0;
+            }
+            catch
+            {
+                throw;
```

Comment:
There used to be a Console.Writeline in catch block for debugging purpose, which was removed. So try/catch here really has no advantage, will remove it.